docs/source/release/v3.14.0/diffraction.rst

```diff
@@ -87,4 +87,13 @@ New
 - :ref:`HB2AReduce <algm-HB2AReduce>` algorithm reduces HFIR POWDER (HB-2A) data
 - :ref:`LoadGudrunOutput <algm-LoadGudrunOutput>` is a new algorithm that allows users to load the standard Gudrun output files into Mantid.
 
+
+Engineering Diffraction
+-----------------------
+
+New
+###
+
+- Scripts added that produce the same results as the engineering gui, this is to allow use with autoreduction.
```

Comment:
Maybe say ISIS engineering gui and ISIS autoreduction. Maybe mention the instruments it'll support (IMAT, ENGINX)